lib/inspec/runner.rb

```diff
@@ -71,6 +77,12 @@ def initialize(conf = {})
       configure_transport
     end
 
+    def cleanse_waivers(waivers)
+      waivers.reject do |h|
+        # h[:expiry] && h[:reason] && h[:id] && etc ...
```

Comment:
I don't think there is any filtering to be done here - it's a list of YAML filenames; unless we exclude a whole file.